tests/index.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-env mocha */
+
+const P = require('bluebird')
+const deleteFolder = P.promisify(require('rimraf'))
+const clearRequire = require('clear-require')
+const assert = require('chai').assert
+const path = require('path')
+
+const generator = require('../src/generator')
+const listFolders = require('../src/lib/list_folders')
+const read = require('../src/lib/read')
+
+const EXAMPLES_PATH = path.join(__dirname, '/../examples/')
+const OUTPUT_PATH = path.join(__dirname, '/../output/')
```

Comment:
Let's not pollute the root folder with rendered test templates, but otherwise this looks fine for now.

Reply:
yeah that's right, i was already thinking about moving this to another place. will do it on the next version